solidus_virtual_gift_card.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.require_path = "lib"
   s.requirements << "none"
 
-  s.add_dependency "solidus", "~> 1.2"
+  s.add_dependency "solidus_core", "~> 1.2"
```

Comment:
I believe so, I haven't verified that those are the only other dependencies, but a quick glance showed that to be the case.